Add a note about using additional-config with the import command (#1477)
neo4j/docs-operations: modules/ROOT/pages/tools/neo4j-admin/index.adoc

The help command also accepts a parameter.
For example, `neo4j help`, `neo4j-admin help`, `neo4j-admin [category] help`, `neo4j-admin help [category]`, `neo4j help [command]`, or `neo4j-admin [category] [command ] help [subcommand]`.

== Limitations

> When using both a multi-value option and a positional parameter, the multi-value option behaves as greedy and pulls in the next positional parameter via its convertor.

Reviewer: ```suggestion
When using both a multi-value option and a positional parameter, the multi-value option is "greedy" and pulls in the next positional parameter via its convertor.
```
